### PR TITLE
PDT easyblock fixes

### PR DESCRIPTION
**Pull request by geimer**
- Actually pass compiler flag to configure
- Allow building with dummy toolchain

### CODE REVIEW

File: easybuild/easyblocks/p/pdt.py [hunk 1]
```diff
@@ -53,14 +53,16 @@ def configure_step(self):
...
             toolchain.GCC: '-GNU',
             toolchain.INTELCOMP: '-icpc',
         }
         comp_fam = self.toolchain.comp_family()
         if comp_fam in known_compilers:
-            self.compiler_opt = known_compilers[comp_fam]
+            compiler_opt = known_compilers[comp_fam]
         else:
             raise EasyBuildError("Compiler family not supported yet: %s" % comp_fam)
+        self.cfg.update('configopts', compiler_opt)
```
Comment (boegel) [inline easybuild/easyblocks/p/pdt.py]: so, `compiler_opt` was not used at all before...?

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyblocks/p/pdt.py (hunk 1 above)
Comment (geimer) [inline easybuild/easyblocks/p/pdt.py]: Apparently, no.  So, one could argue that this flag doesn't matter as the easyblock has worked fine until now, but I can't say for sure which consequences omitting this flag really has. I guess that `configure` always picked up GCC, regardless of the actual toolchain...

Comment (boegel) [inline easybuild/easyblocks/p/pdt.py]: Yeah, I guess so... Nice catch!

**Comment by boegel** [comment]
Tested with existing PDT easyconfigs and the ones from #2928, works great, so going in.

Thanks @geimer!

**Pull request merged**